src/core/pyproject.toml

```diff
@@ -54,7 +55,7 @@ fallback_version = "0.0.0"
 root = "../.."
 
 [project.optional-dependencies]
-dev = ["black", "flake8", "pytest", "pytest-celery", "sqlalchemy-stubs", "schemathesis", "build", "wheel", "setuptools", "setuptools_scm"]
+dev = ["black", "flake8", "pytest", "pytest-celery", "sqlalchemy-stubs", "schemathesis", "build", "wheel", "setuptools", "setuptools_scm", "lxml.html"]
```

Comment:
Since it as a decency needed during runtime, it should not be part of the "optional-dependencies"